.gitignore

```diff
@@ -1,9 +1,9 @@
 # Ignore Visual Studio files
 .vs/
-
+.vscode/
```

Comment:
I'm a little confused here. You are adding .vscode to gitignore but the repo has .vscode/settings.json in it. Does that mean that the team does want to keep track of/share .vscode settings or should .vscode directory be removed from the repo?